src/style.css

```
@@ -1 +1,64 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+}
```

Не залишай пусті рядки між посиланнями
```suggestion
.header {
  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
}
```